code/modules/ruins/lavalandruin_code/elephantgraveyard.dm

```diff
@@ -0,0 +1,227 @@
+
+
+//***Fluff items for lore/intrigue
+/obj/item/paper/crumpled/muddy/fluff/elephant_graveyard
+	name = "Posted Warning"
+	desc = "It seems to be smudged with mud and... oil?"
+	info = "<B>ATTENTION TO WHOOM IT MAY CONCERN</B><BR><BR>This area is property of Nanotrasen Mining Devision.<BR><BR>Tresspassing in this area is illegal, highly dangerous, and subject to several NDAs.<br><br>Please turn back now, under intergalactic law section 48-R."
+
+/obj/item/paper/crumpled/muddy/fluff/elephant_graveyard/rnd_notes
+	name = "Research Findings: Day 26"
+	desc = "Hunh, this one page looks like it was torn out of a full book. How odd."
+	icon_state = "docs_part"
+	info = "<b>Researcher name:</b> B--*--* J--*s.<BR><BR>Detailed findings:<i>Today the camp site's cond-tion has wor--ene*. The ashst--ms keep blocking us off from le-ving the sit* for m-re supplies, and it's lo-king like we're out of pl*sma to p-wer the ge-erat*r. Can't rea-*y study c-*bon *ating with no li--ts, ya know? Da-*y's been going -*f again and ag-*n a-*ut h*w the company's left us to *ie here, but I j-s* keep tell-ng him to stop che*-in* out these damn graves. We m-y b* archeologists, but -e sho*ld have t-e dec-**cy to know these grav-s are *-l NEW.</i><BR><BR><b>The rest of the page is just semantics about carbon dating methods.</b>"
+
+/obj/item/paper/crumpled/muddy/fluff/elephant_graveyard/mutiny
+	name = "Hastily Scribbled Note"
+	desc = "Seems like someone was in a hurry."
+	info = "Alright, we all know that stuck up son a bitch is just doing this to keep us satisifed. Who the hell does he think he is, taking extra rations? We're OUT OF FOOD, CARL. Tomorrow at noon, we're going to try and take the ship by force. He HAS to be lying about the engine cooling down. He HAS TO BE. I'm tellin ya, with this implant I lifted off that last supply ship, I got the smarts to get us offa this shithole. Keep your knife handy carl."
+
+/obj/item/paper/fluff/ruins/elephant_graveyard/hypothesis
+	name = "Research Document"
+	desc = "Standard Nanotrasen typeface for important research documents."
+	info = "<b>Day 9: Tenative Conclusions</b><BR><BR>While the area appears to be of significant cultural importantance to the lizard race, outside of some sparce contact with native wildlife, we've yet to find any reasoning for the nature of this phenomenon exactly. It seems that organic life is communally drawn to this planet as though it functions as a final resting place for intelegent life. As per company guidelines, this site shall be given the following classification: 'LZ-0271 - Elephant Graveyard' <BR><BR><u>Compiled list of Artifact findings (Currently Sent Offsite)</u><BR>Cultist Blade Fragments: x8<BR>Brass Multiplicative Ore Sample: x105<BR>Syndicate Revolutionary Leader Implant (Broken) x1<BR>Extinct Cortical Borer Tissue Sample x1<BR>Space Carp Fossil x3"
```

Comment:
- `importantance` -> `importance`
- `we've yet to find any reasoning for the nature of this phenomenon exactly` -> `we've yet to find any exact reasoning for the nature of this phenomenon`
could also change `we've` to `we're` to make it sound more present tense, since they would be working as this was written
- `intelegent` -> `intelligent`